### ISSUE TITLE
Show related PR(s) in Icon details

### ISSUE THREAD

**Issue by zaaakher** [lucide-icons/lucide#1685]
### Package

- [ ] lucide
- [ ] lucide-angular
- [ ] lucide-flutter
- [ ] lucide-preact
- [ ] lucide-react
- [ ] lucide-react-native
- [ ] lucide-solid
- [ ] lucide-svelte
- [ ] lucide-vue
- [ ] lucide-vue-next
- [ ] Figma plugin
- [ ] all JS packages
- [X] site

### Description

Just a small idea, if there's a button next to the contributors that links to the PR (or multiple PRs) related to that Icon so the we can go to the PR to see the discussion leading to the final decision 

👇here's a quick mockup of how it would look like:
![image](https://github.com/lucide-icons/lucide/assets/46135573/1dbc3779-5d57-4799-b430-b58c68fe7f17)
Notice the PR number next to the contributors

### Use cases

I was going through some of the icons and I noticed that `shield-off` and `shield-ban` have very different contributors and I noticed an inconsistency with the degree of the diagonal line and I was wondering if there was a discussion regarding that decision so I had that idea of linking all PRs related to the icon so people can trace all discussions related to the icon to understand reasons behind certain design decisions.

### Checklist

- [X] I have searched the existing issues to make sure this bug has not already been reported.

**Comment by zaaakher** [comment]
I just noticed I can click on the "last changed" part of the icon

![image](https://github.com/lucide-icons/lucide/assets/46135573/26d934ae-d947-44ef-9b82-70443dd9812f)


which will take me to the release which can lead me to the related PR's and that way.

Perhaps the "last changed" release tag could be added to the icon quick view in one of these areas 👇?
![image](https://github.com/lucide-icons/lucide/assets/46135573/3d13d483-f330-4e56-ba87-cf927756aa95)

**Comment by jguddas** [comment]
What about merging the two detail views into one by creating a scrollable bottom sheet like this?
https://play.tailwindcss.com/tIdRAz5DcD

**Comment by ericfennis** [comment]
@zaaakher Yeah good idea!

**Comment by zaaakher** [comment]
> What about merging the two detail views into one by creating a scrollable bottom sheet like this? https://play.tailwindcss.com/tIdRAz5DcD

Yes that would be great 👍 especially that the X button will close it and I can easily search for another icon

👇 currently in the full page detailed view there's no way to search for another icon, I have to click on the "icons" button in the nav bar to go back
![image](https://github.com/lucide-icons/lucide/assets/46135573/6cceb9ae-9e34-42c8-b4c6-7884cb2c0a3c)

**Comment by zaaakher** [comment]
Or there could be a search bar in the detailed page like this 👇
![image](https://github.com/lucide-icons/lucide/assets/46135573/10a97059-b4b1-4ded-a506-5367eb20a97a)
